src/modlunky2/ui/trackers/pacifist.py

```diff
@@ -48,6 +48,10 @@ def disable_button(self):
 
 
 class PacifistWatcherThread(WatcherThread):
+    def initialize(self):
```

Comment:
Could just make the base class's initialize do nothing rather than requiring a noop initialize in every subclass

Reply:
My concern is it seems easy to implement a stateful tracker and overlook this method. I don't feel strongly, though; I suspect it comes down to which flavor is more likely in the future.

In more dubious ideas... It's intuitively appealing to tie the lifecycle of the specific trackers to the process, with a 'manager' that constructs them when it notices when the game starts/exits . I haven't thought through the details tho.